apps/webstore/src/app/shared/components/items/basket-item/basket-item.component.ts

```diff
@@ -43,32 +43,40 @@ export class BasketItemComponent implements OnInit {
     }
   }
 
-  increaseAmount() {
-    this.item.amount = this.item.amount + 1;
+  increaseQuantity() {
+    this.item.quantity = this.item.quantity + 1;
     this.updatePrice();
   }
 
   increaseSize() {
-    switch (this.item.size) {
-      case '20cm x 20cm':
-        this.item.size = '25cm x 25cm';
+    switch (this.item.dimension) {
+      case 'small':
+        this.item.dimension = DesignDimensionEnum.medium;
         this.updatePrice();
         break;
-      case '25cm x 25cm':
-        this.item.size = '30cm x 30cm';
+      case 'medium':
+        this.item.dimension = DesignDimensionEnum.large;
         this.updatePrice();
         break;
     }
   }
 
   decreaseSize() {
-    switch (this.item.size) {
-      case '25cm x 25cm':
-        this.item.size = '20cm x 20cm';
+    switch (this.item.dimension) {
+      case DesignDimensionEnum.small:
+        this.toastService.showAlert(
```

Comment:
Didn't you say it would be better to not have this option at all? also without the alert?

Reply:
Resolved in 921f3af
When the button is disabled, this doesn't trigger
But currently, we don't disable the button, and thus the user has no feedback that their action is invalid